src/protocols/secure_channel/CASESession.h

```diff
@@ -58,19 +58,21 @@ constexpr size_t kCASEResumptionIDSize = 16;
 #define CASE_EPHEMERAL_KEY 0xCA5EECD0
 #endif
 
-struct CASESessionSerialized;
+struct CacheableCASEParameters;
 
-struct CASESessionSerializable
+struct CASESessionCachable
 {
-    uint8_t mVersion;
     uint16_t mSharedSecretLen;
     uint8_t mSharedSecret[Crypto::kMax_ECDH_Secret_Length];
     uint16_t mMessageDigestLen;
```

Comment:
Let's remove the message digest from this structure.

Reply:
Done